src/language/interpreter.rs

```diff
@@ -1091,17 +1091,23 @@ mod tests {
     /// Creates a benchmark test for the interpreter
     /// The test does the following:
     ///  1. Parses $glenside_str as glenside expression
-    ///  2. Creates a new Environment from the vector of (key, value) pairs
-    ///  3. Calls the interpreter with the glenside expression and env,
-    ///         and passes the value to check_correct
+    ///  2. Creates a new Environment from the vector of (key, value) pairs if
+    ///     present
+    ///  3. Calls the interpreter with the glenside expression and env, and
+    ///         passes the value to check_correct
```

Comment:
Can you make the second line indentation here consistent? I don't really care which you use, honestly!